pkg/proxy/iptables/proxier.go

```diff
@@ -1071,6 +1071,7 @@ func (proxier *Proxier) syncProxyRules() {
 				// If/when we support "Local" policy for VIPs, we should update this.
 				writeLine(proxier.natRules, proxier.localDetector.JumpIfNotLocal(args, string(KubeMarkMasqChain))...)
 			}
+			args[1] = string(kubeServicesChain)
```

Comment:
so, args[1] is the chain name, and we replace it here ... you should add a comment explaining it

Reply:
ack, will leave a comment.